tidy3d/components/geometry/primitives.py

```diff
@@ -166,6 +166,19 @@ def _surface_area(self, bounds: Bound) -> float:
         return area
 
 
+class Point(Sphere):
```

Comment:
Does this need to inherit from `Sphere`? or can we just define fixed points as `list[Coordinate]` to keep it simple? `Point` would add another class for users to learn.

Reply:
If we keep `Point`, be sure to add to the API reference `api.rst`. but I'd probably rather use `Coordinate` tuples

Reply:
Good point, I'll change to use Coordinate